agent/src/test/scala/za/co/absa/atum/agent/model/MeasuresBuilderTest.scala

```diff
@@ -50,7 +49,7 @@ class MeasuresBuilderTest extends AnyFlatSpecLike {
       MeasureDTO("unsupportedMeasure", Seq("col"))
     )
 
-    assertThrows[MeasureException](MeasuresBuilder.mapToMeasures(unsupportedMeasure))
+    assert(MeasuresBuilder.mapToMeasures(unsupportedMeasure).isEmpty)
```

Comment:
see line 47, the test description needs to ba changed as well

Reply:
ok, reformulated